--- libglade-0.17/configure.in.orig	2001-09-07 05:00:29.000000000 +0200
+++ libglade-0.17/configure.in	2026-03-15 12:00:00.000000000 +0100
@@ -91,14 +91,12 @@
 dnl gettext stuff ... there is no message catalog for libglade -- libglade
 dnl provides translation for the XML files it reads in.
 ALL_LINGUAS=""
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT([external])
 
 AC_CONFIG_FILES([Makefile
 libglade.spec
 macros/Makefile
-intl/Makefile
 glade/Makefile
-doc/Makefile
 libglade-config
 libgladeConf.sh
 libglade.pc
--- libglade-0.17/Makefile.am.orig	2001-09-07 05:00:13.000000000 +0200
+++ libglade-0.17/Makefile.am	2026-03-15 12:00:00.000000000 +0100
@@ -43,7 +43,7 @@
 bin_SCRIPTS = libglade-config libglade-xgettext
 noinst_PROGRAMS = test-libglade
 test_libglade_LDADD = $(THE_LIBS) $(XML_LIBS)
-SUBDIRS = intl macros glade doc
+SUBDIRS = po macros glade
 
 m4datadir = $(datadir)/aclocal
 m4data_DATA = libglade.m4
