--- libglade-0.17/doc/Makefile.am~	Fri Sep  7 05:13:48 2001
+++ libglade-0.17/doc/Makefile.am	Tue Oct  2 02:14:18 2001
@@ -42,6 +42,7 @@
 	    LD="$(LIBTOOL) --mode=link $(CC)" \
 	    CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" \
 	        gtkdoc-scanobj --module=$(DOC_MODULE) \
+			--nogtkinit \
 	                --types=$(srcdir)/$(DOC_MODULE).types \
 	                --output-dir=$(srcdir)
 	cd $(srcdir) \
